docs/objetivo-4.md

```diff
@@ -0,0 +1,13 @@
+# Documentación hito 4
+
+## Elección del gestor de tests: pytest
+
+Las posibles opciones eran Robot(el más popular), PyTest, UnitTest y DocTest. Robot a pesar de ser muy completo tenía demasiadas funcionalidades que no se le darán uso en este proyecto y DocTest presenta una sintaxis muy fea que no me acaba de convencer. Entre UnitTest y PyTest elegí PyTest debido a que tiene una mayor popularidad y está muy depurado.

```

Comment:
Los criterios tenéis que establecerlos de antemano. Y por lo menos enlazar lo que decís. Unittest no es un gestor de tests.